builddb.py

```diff
@@ -5,7 +5,7 @@
 import uuid
 
 # Reset database
-db.drop_all()
+#db.drop_all()
```

Comment:
Can probably just remove this line from .git

Reply:
builddb.py is only called during database reconstruction. no harm keeping this line.